app/src/main/java/com/b_lam/resplash/ui/photo/detail/PhotoDetailActivity.kt

```diff
@@ -257,8 +272,10 @@ class PhotoDetailActivity :
                 val downloadManagerWrapper: DownloadManagerWrapper by inject()
                 viewModel.downloadId = downloadManagerWrapper.downloadPhoto(url, photo.fileName)
             } else {
-                viewModel.downloadUUID = DownloadWorker.enqueueDownload(applicationContext,
-                    DownloadAction.DOWNLOAD, url, photo.fileName, photo.id)
+                viewModel.downloadUUID = DownloadWorker.enqueueDownload(
+                    applicationContext,
+                    DownloadAction.DOWNLOAD, url, photo.fileName, photo.id
+                )
```

Comment:
Can be formatted to:
```suggestion
                viewModel.downloadUUID = DownloadWorker.enqueueDownload(
                    applicationContext,
                    DownloadAction.DOWNLOAD,
                    url,
                    photo.fileName,
                    photo.id
                )
```